clang5/plan.sh

```diff
@@ -2,14 +2,14 @@ source "$(dirname "${BASH_SOURCE[0]}")/../clang/plan.sh"
 
 pkg_name=clang5
 pkg_origin=core
-pkg_version=5.0.1
+pkg_version=5.0.2
```

Comment:
@TheDarkula we actually deprecated clang5 https://github.com/habitat-sh/core-plans/pull/2239

However, were you able to build clang5 by going to 5.0.2?

Reply:
Fixed :)
I'm fairly certain it built (don't exactly remember at the moment)